system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php

```diff
@@ -6,20 +6,22 @@
 						<?php
 						$version_class = 'app-about__version';
 						$update_available = isset($new_version);
+						$update_is_major_version = false;
 						$vital_update = $update_available && $new_version['security'];
 
-						if ( ! empty($version_identifier))
-						{
+						if ( ! empty($version_identifier)) {
 							$version_class .= ' app-about__version--dev';
-						}
-						elseif ($update_available)
-						{
-							if ($vital_update)
-							{
-								$version_class .= ' app-about__version--update-vital';
+						} elseif ($update_available) {
+							$version_major = explode('.', APP_VER, 2)[0];
+							$update_version_major = explode('.', $new_version['version'], 2)[0];
+
+							if (version_compare($version_major, $update_version_major, '<')) {
+								$update_is_major_version = true;
```

Comment:
I think we need to make sure to not apply "vital" class even if update is vital, as that is only relevant per-major-release

Reply:
I've modified this to be an elseif so it won't show the vital update styles if there's already a major update available.

Reply:
I think this is good. Let's package and test it!